blueprints.csv

```diff
@@ -46,7 +46,7 @@ CuredSpinefish,0,
 DoubleTank,0,
 RadiationSuit,0,
 ReinforcedDiveSuit,0,
-StillSuit,0,
+StillSuit,1,
```

Comment:
Since the databox is not being picked up currently, just remove this from the pool of other things